client/blocks/login/login-form.jsx

```diff
@@ -166,15 +160,11 @@ export class LoginForm extends Component {
 		const isOauthLogin = !! oauth2Client;
 		let signupUrl = config( 'signup_url' );
 
-		if ( isOauthLogin ) {
-			if ( config.isEnabled( 'signup/wpcc' ) ) {
-				signupUrl = '/start/wpcc?' + qs.stringify( {
-					oauth2_client_id: oauth2Client.id,
-					oauth2_redirect: redirectTo
-				} );
-			} else {
-				signupUrl = oauth2Client.signupUrl;
-			}
+		if ( isOauthLogin && config.isEnabled( 'signup/wpcc' ) ) {
```

Comment:
We should do something about this feature, either remove this check here or set `signup/wpcc: true` in all envs

Reply:
Oh it's handled in #17530 nvm then